src/OpenApi/src/Services/Schemas/OpenApiSchemaStore.cs

```diff
@@ -17,16 +19,39 @@ internal sealed class OpenApiSchemaStore
     private readonly ConcurrentDictionary<OpenApiSchemaKey, JsonObject> _schemas = new()
```

Comment:
Some similar code [here](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/a97c53fd4dfe1a2ca2e2607959cf6c002b02c2e0/src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/JsonSerializerDataContractResolver.cs#L248-L276) in case that's useful too.